### PR TITLE
ShareX installation guide

### PR DESCRIPTION
**Pull request by Spacelord09**
ShareX is a windows tool to upload and share screenshots easily.

Since I have little experience with RST, I would ask for understanding for any errors that may exist.
In addition, I ask you to look for spelling fails and to report them to me.

Thank you ;D

### CODE REVIEW

**Comment by Spacelord09** [comment]
Why is the code block broken in step 3?

**Comment by Spacelord09** [comment]
@luto can you help me? ;D

File: source/guide_sharex.rst [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+------
+Open ShareX and navigate to "**Destinations** > **Destinations settings** > **FTP / FTPS / SFTP**".
+Now click "**Add**" to create a new destination for our Pictures.
+
+Step 3
+------
+Now write the following in the coresponding fields.
+
+::
+  Name: IMG Uberspace
```
Comment (luto) [inline source/guide_sharex.rst]: add an empty line after `::` to fix the code block.

**Comment by Spacelord09** [comment]
And how to fix the picture overlapping with the info block? Is my spelling right?

**Comment by Spacelord09** [comment]
Done?

**Comment by Spacelord09** [comment]
@luto Passt jetzt alles?

**Comment by luto** [comment]
Danke für den PR! :) Ich hab noch keinen genauen Blick drauf geworfen.

**Comment by Spacelord09** [comment]
@luto So. Ich habe die commits messages nochmal geändert, die commits gesquasht und den Vorgaben angepasst. Ich hoffe auf einen Ubermug ;D

**Comment by luto** [comment]
Thank you :)

**Comment by luto** [comment]
Please ping hallo@uberspace.de about your cup! Normally we send out mails, but there is no mail address in your commits :)

**Comment by Spacelord09** [comment]
@luto You can use my mail address in the guide😉

**Pull request merged**